content/docs/scaffolder/custom-fields/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Custom Scaffolder Field Extensions
+publishedDate: '2024-02-28'
+description: How to develop and register custom Scaffolder Field Extensions 
+---
+
+You can extend and customize the available Scaffolder functionality by using [Roadie Custom Plugins](/docs/custom-plugins/overview). You can use the same workflow and development experience to build your own Custom Scaffolder Field Extensions as any other plugin, using the Roadie CLI.
+
+To see more information on how to develop custom plugins. It is recommended to navigate to the [Custom Plugins Documentation](/docs/custom-plugins/getting-started/). To construct the correct type of React components that are compatible to be registered as Scaffolder Field Extensions, the recommended approach is to [follow the tutorial on the Open Source Backstage documentation page.](https://backstage.io/docs/features/software-templates/writing-custom-field-extensions/)
```

Comment:
`### Constructing React components for your Field Extension`

To construct the correct type of React components that are compatible to be registered as Scaffolder Field Extensions, the recommended approach is to [follow the tutorial on the Open Source Backstage documentation page.](https://backstage.io/docs/features/software-templates/writing-custom-field-extensions/)

Reply:
please use the suggestion feature on GitHub for these. Much easier to handle